src/utils/validationUtil.ts

```diff
@@ -1,5 +1,6 @@
 import { ValidationResult, any, string, validate } from 'joi';
 import { isDate } from 'lodash';
+import { DefectGETIVA } from '@dvsa/cvs-type-definitions/types/iva/defects/get';
```

Comment:
Can we alias this to something like IVADefect? it won't be being used as a GET within this service.